components/evaluate-spl.yaml

```diff
@@ -47,52 +49,79 @@ implementation:
 
       cd workdir/${MAKEDIR}
 
+      if test "${s3_database_dir}" != None ; then
+        aws s3 sync --no-progress ${s3_database_dir} ./database/ --exclude '*'  --include '*type2id.json' --include '*pretrained_bert_models*' --include '*entity_mappings/config.json' --include '*entity_mappings/qid2title.json' --include '*entity_mappings/qid2eid.json' --include '*canonical2type.json' --include '*es_*.json' --include '*wikidatatitle_to_typeid_0905.json' --include '*wikidataqid_to_bootlegtypeid.json' --include '*entityQID_to_wikidataTypeQID.json' --include '*alias2qids_wiki*' --include "*${bootleg_model}/*.json" --include "*${bootleg_model}/*.pt"
+      fi
+
       IFS='+'; read -ra PLS <<<"$pred_languages"; IFS=' '
+      IFS='+'; read -ra ES <<<"$eval_set"; IFS=' '
+
       echo "${PLS[@]}"
 
       export TZ=America/Los_Angeles
+      export GENIENLP_DATABASE_DIR=./database/
```

Comment:
You're going to make this change to the regular evaluate job too right?

Reply:
Yeah. Once we figure out the bootleg part, I will make an end2end pipeline with no spl in the middle!